include/minirest_http.hrl

```diff
@@ -0,0 +1,54 @@
+-define(CONTINUE, <<"100">>).
```

Comment:
Please add header guards to avoid errors on double including of the header

Reply:
Fixed. Thank you for your suggestion!